CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Reporting issues
+
+Before reporting a bug, please take a look at the [FAQ](https://github.com/abo-abo/swiper/blob/master/README.md), as well as the [manual](http://oremacs.com/swiper/) and the [wiki](https://github.com/abo-abo/swiper/wiki). Please also make sure that there is not yet an existing issue.
+
+In order to fix it, we need to be able to reproduce the bug you encountered as closely as possible, so please describe the problem as detailed as you can. It is important that we can rule out interference with other emacs packages or customisations, therefore use `emacs -Q` to bypass your init file or use the shell code below to sandbox your emacs environment even further.
```

Comment:
Couple of nitpicks:
* "describe the problem in as much detail as possible" instead of "describe the problem as detailed as you can"
* "Emacs packages" instead of "emacs packages"